doc/dialects/exasol.md

```diff
@@ -84,6 +84,34 @@ USING SCHEMA_FOR_VS_SCRIPT.ADAPTER_SCRIPT_EXASOL WITH
     EXA_CONNECTION  = 'EXA_CONNECTION';
 ```
 
+#### Auto-generated Datatype Mapping Using EXA Import
+
+Using `IMPORT FROM EXA` might lead to some unexpected datatype mappings. Unlike for a JDBC connection there's no explicit data mapping being generated when using `IMPORT FROM EXA`. As a solution the Exasol Virtual Schema in version 7.2.0 and later supports parameter `GENERATE_JDBC_DATATYPE_MAPPING_FOR_EXA`:
```

Comment:
See my comment above about the parameter name.
Maybe we should simply say "map datatypes" instead of "generate a mapping". 
This is also shorter and avoids the questions raised by "generating something". 
I am also thinking of a term like "infer".

Reply:
Proposal:

> Unlike for a JDBC connection `IMPORT FROM EXA` does not use an explicit datatype mapping.  In consequence columns of type `HASHTYPE` are mapped to `VARCHAR` and joining such a column therefore failed in Exasol 7.1 with error message `Feature not supported: Incomparable Types: VARCHAR(32) UTF8 and HASHTYPE(16 BYTE)!`.
> 
> Exasol Virtual Schema in version 7.2.0 and later mitigates this by offering parameter `GENERATE_JDBC_DATATYPE_MAPPING_FOR_EXA`.